.github/workflows/open-pr-on-changes.yaml

```diff
@@ -16,6 +16,7 @@ jobs:
           { project: GNOME Shell, repo: "https://gitlab.gnome.org/GNOME/gnome-shell.git", branch: master, projectdir: gnome-shell, src: data/theme/, dest: gnome-shell/upstream },
           { project: GNOME Shell Resources, repo: "https://gitlab.gnome.org/GNOME/gnome-shell.git", branch: master, projectdir: gnome-shell, src: data/gnome-shell-theme.gresource.xml, dest: gnome-shell/upstream/data/gnome-shell-theme.gresource.xml },
           { project: GTK, repo: "https://gitlab.gnome.org/GNOME/gtk.git", branch: gtk-3-24, projectdir: gtk, src: gtk/theme/Adwaita/, dest: gtk/upstream/gtk+3.0/Adwaita },
+          { project: Libhandy, repo: "https://gitlab.gnome.org/GNOME/libhandy.git", branch: master, projectdir: gtk, src: src/themes/, dest: gtk/upstream/libhandy/Adwaita },
```

Comment:
This is the name of the cloned project, so it is probably "libhandy"

Reply:
For symmetry, I would keep the same name of the dest dir "gtk/upstream/libhandy/themes"